Source/Generic/ECCommandLineEngine.m

```diff
@@ -397,7 +397,13 @@ - (void)outputError:(NSError *)error format:(NSString *)format, ...
 	NSString* string = [[NSString alloc] initWithFormat:format arguments:args];
 	va_end(args);
 
-	NSString* errorString = error ? [NSString stringWithFormat:@"(%@ %@:%ld)\n", error.localizedFailureReason, error.domain, error.code] : @"";
+
+
+	NSString* errorString = @"";
```

Comment:
Minor change to prevent (null) from being logged if the error has no failure reason.